[Detection rules] Add 7.16.0 updates (#1283)
elastic/security-docs: docs/detections/prebuilt-rules/rule-details/threat-intel-filebeat-module-indicator-match.asciidoc

* SecOps
* Monitoring

> *Version*: 3 (<<threat-intel-filebeat-module-indicator-match-history, version history>>)

Reviewer: Version incremented from 1 to 3

These kinds of questions can help understand if the activity is related to legitimate behavior.
- Consider the user and their role within the company, is this something related to their job or work function?

### False Positive Analysis
- For any matches found, it's important to consider the initial release date of that indicator. Threat intelligence can
be a great tool for augmenting existing security processes, while at the same time it should be understood that threat
intelligence can represent a specific set of activity observed at a point in time. For example, an IP address
may have hosted malware observed in a Dridex campaign six months ago, but it's possible that IP has been remediated and
no longer represents any threat.
- Adversaries often use legitimate tools as network administrators such as `PsExec` or `AdFind`, these tools often find their
way into indicator lists creating the potential for false positives.
- It's possible after large and publicly written campaigns, curious employees might end up going directly to attacker infrastructure and generating these rules

### Response and Remediation

> - If suspicious or malicious behavior is observed, immediate response should be taken to isolate activity to prevent further

Author: ```suggestion
- If suspicious or malicious behavior is observed, take immediate action to isolate activity to prevent further
```

post-compromise behavior.
- One example of a response if a machine matched a command and control IP address would be to add an entry to a network
device such as a firewall or proxy appliance to prevent any outbound activity from leaving that machine.
- Another example of a response with a malicious file hash match would involve validating if the file was properly quarantined,

> review current running processes looking for any abnormal activity, and investigating for any other follow-up actions such as persistence or lateral movement

Reviewer 2: ```suggestion
reviewing current running processes for any abnormal activity, and investigating for any other follow-up actions such as persistence or lateral movement.
```
